tutorials/single-page-tutorials/nested-nfts-with-wallet-lib.md

```diff
@@ -217,12 +221,197 @@ Run the script `create-nfts.js` and check the console output to follow along the
 node create-nfts.js
 ```
 
-# Open ToDo's:
+Since you were minting three NFTs, your console output should look like this:
+
+```console
+Start local IPFS node for upload:
+<IPFS-node-swarm-information>
+
+
+Your file "avatar.jpg" was successfully uploaded to IPFS:
+https://ipfs.io/ipfs/<ipfs-content-identifier>
+
+Your NFT was successfully minted in this block:
+https://explorer.shimmer.network/testnet/block/<blockId>
+
+
+
+Your file "mask.jpg" was successfully uploaded to IPFS:
+https://ipfs.io/ipfs/<ipfs-content-identifier>
+
+Your NFT was successfully minted in this block:
```

Comment:
For me it doesn't mint the second NFT.
It panics with 
```
Error:  {
  type: 'ClientError',
  error: "`The wallet account doesn't have enough balance. It only has 0, required is 60900`"
}
```

Not sure why tho. I also tired to add an additional sync() 🤔

Reply:
I added a function to await transaction confirmation before the next NFT is minted. Probably not the most elegant way, but gets the job done.